metrics/github/cli.py

```diff
@@ -69,10 +70,12 @@ def pr_throughput(ctx, org, date):
 
     with TimescaleDBWriter(GitHubPullRequests) as writer:
         opened_prs = api.prs_opened_on_date(org, date)
+        opened_prs = drop_archived_prs(opened_prs)
         log.info("%s | %s | Processing %s opened PRs", date, org, len(opened_prs))
         process_prs(writer, opened_prs, date, name="prs_opened")
 
         merged_prs = api.prs_merged_on_date(org, date)
+        merged_prs = drop_archived_prs(merged_prs, key="merged")
```

Comment:
I don't see the need for this filter. There won't be any newly-merged PRs for an archived repo, will there?